ambry-clustermap/src/main/java/com/github/ambry/clustermap/ClusterMapUtils.java

```diff
@@ -96,6 +96,9 @@ public class ClusterMapUtils {
   static final long MIN_REPLICA_CAPACITY_IN_BYTES = 1024 * 1024 * 1024L;
   static final long MAX_REPLICA_CAPACITY_IN_BYTES = 10L * 1024 * 1024 * 1024 * 1024;
   static final long MIN_DISK_CAPACITY_IN_BYTES = 10L * 1024 * 1024 * 1024;
+  // TODO: Temporary defaults to be used when adding replicas in helix FULL_AUTO mode. These replica configs will need
+  //  to stored in configs or helix for each cluster.
+  static final long DEFAULT_REPLICA_CAPACITY_IN_BYTES = 100L * 1024 * 1024 * 1024;
```

Comment:
it should be in the helix cluster config. Each partition would have a weight, this weight is determined by the replica capacity. We should include the partition weight and the replica capacity in the same cluster config.

Reply:
Sure. I will add these configs in subsequent PR and remove these default values.